grouper/entities/permission_grant.py

```diff
@@ -1,3 +1,10 @@
 from typing import NamedTuple
 
 PermissionGrant = NamedTuple("PermissionGrant", [("name", str), ("argument", str)])
```

Comment:
Thoughts on removing this entirely? The only place it's currently used is `permission_grants_for_user`, where it could be replaced by `GroupPermissionGrant` with the added bonus of making it clearer where the grant comes from.

Reply:
Good point! I'll make this change before merging.

Reply:
Oh, now I remember why I didn't do this. It turns out to cause serious performance problems for the SQL version of `permission_grants_for_user`, which I was trying hard to make extremely fast. To include the group names in a naive way requires a SQL query per permission grant the user has, which could be 50x or 100x the number of SQL queries that are otherwise required. There may be some more sophisticated subquery approach that would be faster, but it was complex-enough SQL that I wasn't able to work out a good way to do it.